pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,30 +3,30 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
     "Microsoft.CSharp": "4.0.1",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0-beta-24415-01",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0-preview1-24508-06",
     "Microsoft.VisualBasic": "10.0.1",
     "NETStandard.Library": "1.6.1-beta-24415-01",
     "System.Buffers": "4.0.0",
     "System.Collections.Immutable": "1.2.0",
     "System.ComponentModel": "4.0.1",
     "System.ComponentModel.Annotations": "4.1.0",
     "System.Diagnostics.DiagnosticSource": "4.0.0",
-    "System.Diagnostics.Process": "4.1.1-beta-24415-01",
+    "System.Diagnostics.Process": "4.3.0-preview1-24508-08",
     "System.Dynamic.Runtime": "4.0.11",
```

Comment:
These wouldn't be updated in (e.g.) corefx because they're stable, but [core-setup's logic](https://github.com/dotnet/core-setup/blob/c7e704a7bf5f53080b53e409f821bfbf7f4dc225/build_projects/update-dependencies/UpdateFilesTargets.cs#L129) doesn't skip over stable versions so they should all be auto-updated.